schemas/styles/csl.rnc

```diff
@@ -507,11 +531,28 @@ div {
     
     ## Use to format individual name parts (e.g., "Jane DOE").
     element cs:name-part {
-      attribute name { "family" | "given" },
+      attribute name { "family" | "given" | "alternate" | "literal"},
       affixes,
       font-formatting,
       text-case
     }
+  name.name-type =
+    
+    ## Use to apply separate formatting when a variable contains only personal
+    ## names or a combination of personal and non-personal names ("names") versus
+    ## contains only non-personal names.
+    element cs:name-type {
+      attribute name { "personal" | "non-personal" },
```

Comment:
I don't really have a super-strong opinion here, but more common names for this are "corporate" (this is language from the library world) or "organizational"?

Reply:
Don't have a strong opinion either. My thought was "Is something like 'Committee on Personality Disorders' or 'The Reproducibility Project' really 'organizational' or 'institutional'?" I think it would probably be fine to go with "organizational". I think "corporate" is a little too jargon-y.